roles/ceph-grafana/tasks/configure_grafana.yml

```diff
@@ -1,4 +1,13 @@
 ---
+- name: install ceph-grafana-dashboards package on RedHat or SUSE
+  package:
+    name: ceph-grafana-dashboards
```

Comment:
The issue with this is that it doesn't work for containerized deployment because the ceph repository aren't configured in this scenario

Reply:
According to this [commit](https://github.com/ceph/ceph-container/commit/0de5b43c31a06eee40c88c2677879743c37b0b00) seems we have the package in nautilus containers, hence I'm not sure this task is needed anymore